src/module/vue/components/progress/progress-embed.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
this is one of the components that i'm going to defer for the time being. i'm tempted to just leave it in place for the time being, though.